lib/praxis/resource_definition.rb

```diff
@@ -66,7 +66,7 @@ def media_type(media_type=nil)
       def version(version=nil, options=nil)
         return @version unless version
         @version = version
-        @version_options = options || {using: [:header,:params]}
+        @version_options = options || ApiDefinition.instance.info(version).version
```

Comment:
We need a deprecation warning if somebody passes explicit :using options into the resource.